source/index.html.md

```diff
@@ -57,49 +52,191 @@ code:<AUTH_CODE>
+
+
 
 # API
 The Molo partner API can be used by any partner who has obtained an access token.
 You can access these endpoints by using the `bearer token` authentication header and entering this token as the value.
 
-## Mortgage applications
-Send a GET request to https://partner.molofinance.com/v1/applications/ to see a list of current applications for a user
 
+## Create an application
 ```shell
-curl -X GET \
-  https://partner.molofinance.com/v1/applications/ \
-  -H 'Authorization: Bearer <ACCESS TOKEN>' \
-  -H 'cache-control: no-cache'
+curl
+```
+**Send a `POST` request to https://partner.molofinance.com/v1/applications/ to create a new application.**
+You will need to provide the following parameters:
+
+- `property_value` A decimal value with 2 decimal places. This value is presumed to be in GBP
+- `monthly_rent` A decimal value with 2 decimal places. This value is presumed to be in GBP
+- `desired_loan_amount` A decimal value with 2 decimal places. This value is presumed to be in GBP
+- `loan_term` An integer number of years
+- `mortgage_type` The type of mortgage the customer wants. A choice of:
+    - "new_purchase"
+    - "remortgage_current"
+    - "remortgage_and_borrow"
+- `borrower_type` The type of borrower, a choice of:
+    - "individual_borrower"
+    - "ltd_company"
 
+**Send a `PATCH` request to https://partner.molofinance.com/v1/applications/{pk}/ to update an application.**
+- RESPONSE AND FIELDS TBC
 
+**Send a `GET` request to https://partner.molofinance.com/v1/applications/ to retrieve a list of past applications.**
+- RESPONSE TBC
+
+**Send a `GET` request to https://partner.molofinance.com/v1/applications/{pk}/ to retrieve details of a specific application.**
+- RESPONSE TBC
+
+
+## Create an applicant
+```shell
+curl
+```
+**Send a `POST` request to https://partner.molofinance.com/v1/applicants/ to create a new applicant.**
+You will need to provide the following parameters:
+
+- `application_id` The ID of the application this applicant should be attached to.
+- `is_lead` Is this applicant the lead applicant on this application? Each application requires one and only one lead applicant. A boolean
+- `taxable_income` A decimal value with 2 decimal places. This value is presumed to be in GBP
+- `email` The customer's email address
+- `title` The customer's title, one of **(Do we really need this?)**
+    - "mr"
+    - "mx"
+    - "mrs"
+    - "miss"
+    - "ms"
+    - "sir"
+    - "dr"
+    - "lady"
+    - "lord"
+    - "other"
+- `first_name` The customer's first name
+- `middle_name` The customer's middle name
+- `last_name` The customer's last name
+- `date_of_birth` The customer's date of birth, in ISO 8601 format (yyyy-mm-dd)
+- `mobile` The customer's mobile number in the form "+441615676878"
+- `address` An address object representing the customer's address, in the form:
+    - `address_type` The customer's address type, a choice of:
+        - "current"
+        - "previous"
```

Comment:
Are you sure that 2 addresses type are available? For getting a DIP we require to have at least `current` address type

Reply:
If `address` is not a list, but a single project - we require to have `current` address type

Reply:
I thought they needed to give us their address history for a period of ?3 years or something?
I'm imagining this as a list which we'll do some validation on

Reply:
I'll just make this endpoint accept a single address object and have it default to `current` then